src/scripts/main.js

```diff
@@ -1,3 +1,253 @@
 'use strict';
 
-// write your code here
+const fieldRows = document.querySelectorAll('.field_row');
+const messageWin = document.querySelector('.message_win');
+const messageLose = document.querySelector('.message_lose');
+const button = document.querySelector('.start');
+const messageStart = document.querySelector('.message_start');
+const gameScore = document.querySelector('.game_score');
+
+const GRID_SIZE = 4;
+
+document.addEventListener('keydown', (e) => {
+  switch (e.key) {
+    case 'ArrowUp':
+      grid.slideUp();
+      break;
+    case 'ArrowLeft':
+      grid.slideLeft();
+      break;
+    case 'ArrowDown':
+      grid.slideDown();
+      break;
+    case 'ArrowRight':
+      grid.slideRight();
+      break;
+  }
+});
```

Comment:
use object:
{
  ArrowLeft: 'ArrowLeft',
  ...
}

It's a good practice to avoid mistakes

Reply:
all done